nx/lib/nx.ex

```diff
@@ -7142,6 +7142,42 @@ defmodule Nx do
     impl!(tensor).sort(tensor, tensor, axis: axis, comparator: comparator)
   end
 
+  @doc """
+    Solve the equation a x = b for x, assuming a is a triangular matrix.
+
+  # Examples
+
+  iex> Nx.triangular_solve([[3, 0, 0, 0], [2, 1, 0, 0], [1, 0, 1, 0], [1, 1, 1, 1]], [4, 2, 4, 2])
```

Comment:
Please indent all examples with four spaces.

Reply:
@josevalim will do, I believe `mix format` will do this for me correct?

Reply:
No, it doesn’t change doctests at the moment.